src/platform/site-wide/accessible-VCL-modal.js

```diff
@@ -45,8 +42,18 @@ export function addFocusBehaviorToCrisisLineModal() {
     }
   }
 
-  document.body.addEventListener('click', setOpenControl);
+  function resetFocus() {
+    openControl.focus();
+  }
+
+  triggers.forEach(trigger => {
+    trigger.addEventListener('click', () => {
+      openControl = trigger;
+    });
+  });
```

Comment:
Maybe it's just me, but it took me a moment to understand how this works with the focus. It might be worth adding a comment or a named function to help explain.